core/domain/exp_domain.py

```diff
@@ -285,6 +285,9 @@ def __init__(self, change_dict):
         elif self.cmd == CMD_CREATE_NEW:
             self.title = change_dict['title']
             self.category = change_dict['category']
+        elif self.cmd.endswith('revert_version_number'):
```

Comment:
Why not an equality check?

Reply:
This is AUTO commit so emtire string would be 'AUTO_revert_version_number'. AUTO is constant used for all commits which Oppia issues on behalf of creator. That's why I am not doing equality check as prefix AUTO can change in future

Reply:
I don't think the prefix AUTO can change. It's already part of lots and lots of backend change log models. So an equality check should be fine here really.

That said, this really needs to get merged so that @tjiang11 can run the release, so I'll do that. But it'd be nice to make a constant somewhere for CMD_AUTO_REVERT_VERSION_NUMBER and refer to that where it's used, including here. Could you do that in a separate PR please? I'll file an issue.